scripts/app_load_params_lint.py

```diff
@@ -49,6 +49,12 @@ def check_database_appnames(database_path: Path):
             print(f"[ERROR] Conflict on appName between '{db_rev_variant}' and '{variant}'")
             ret = -1
 
+    app_names_list =[v["appName"].lower().replace(" ","").replace("_", "") for v in
```

Comment:
IF we want to check ` ` and `_`, we should also check `-`

Reply:
Do we really want to check `_` though ?
Do we ban the name `z_app` if `zapp` exists for example ?
Maybe to check with product ?

Reply:
I guess if such conflict happen, it's probably because we want to rename an app rather than creating a new one.

Reply:
check of `-` added